index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="header">
+        <span class="logo-nombre">
+            Gustavo Porcilio
+        </span>
+        <div class="hamburger">
+            <nav class="navlinks">
+                <a href="#porfolio">PORFOLIO</a>
+                <a href="#about">ABOUT</a>
+                <a href="#contact">CONTACT</a>
+            </nav>
+        </div>
+
+    </header>
+    <section class="principal">
+        <div>
+            <h1 class="tittle">Hey There! I'm Gustavo</h1>
+        </div>
+        <div class="profession">
+            <h2>I'm a software developer</h2>
+        </div>
+        <div class="text">
+            <p class="presentation">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.
+            </p>
+        </div>
+        <div class="icons">
+            <a href="https://github.com/MarcoAurelioAntonio" class="github"></a>
+            <a href="https://twitter.com/gustavoporcilio" class="twitter"></a>
+            <a href="https://www.linkedin.com/in/gustavo-porcilio-4496a223a/" class="linkedin"></a>
+        </div>
+    </section>
```

Comment:
- [ ] Make sure your design sticks as much as possible to the one from Figma. You try to update the **backgroung-image** position, fonts on some text, add all elements of the section(icons),... Check everything and make sure you stick your design to the one from Figma.

| Figma Design | Your Design |
| ------------------ | ----------------- |
| ![Screenshot from 2022-07-02 02-26-10](https://user-images.githubusercontent.com/83042742/176980372-d00f6f92-3880-4922-865e-ed3ac0bff990.png) | ![Screenshot from 2022-07-02 02-41-14](https://user-images.githubusercontent.com/83042742/176980344-f1ba25d8-ad80-4198-a114-209dd0f49ca8.png) |